### PR TITLE
[ADD] tox implementation

### PR DESCRIPTION
**Pull request by Laurent-Corron**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  check_readme
+  pre_commit
+
+[testenv]
+skip_missing_interpreters = True
+usedevelop = True
+commands = pytest --verbose --flake8 --cov=loog --cov-branch {posargs}
+deps =
+  pytest
+  pytest-flake8
```
Comment (sbidoul) [inline tox.ini]: pytest-flake8 is not necessary since pre-commit does it

File: tox.ini [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+[tox]
+envlist =
+  py37
+  check_readme
+  pre_commit
+
+[testenv]
+skip_missing_interpreters = True
+usedevelop = True
+commands = pytest --verbose --flake8 --cov=loog --cov-branch {posargs}
```
Comment (sbidoul) [inline tox.ini]: not sure if it must not be `--cov=src/look` ?
also you need to remove `--flake8` now

File: tox.ini [hunk 3]
```diff
@@ -0,0 +1,31 @@
+
+[tox]
+envlist =
+  py37
```
Comment (sbidoul) [inline tox.ini]: add py36 and py38

File: tox.ini (hunk 2 above)
Comment (Laurent-Corron) [reply tox.ini]: Both --cov=src/loog or --cov=loog work.
As for the flake8 flag , my bad, i should have checked.

**Pull request merged**